zproject/settings.py

```diff
@@ -126,6 +126,7 @@ def get_secret(key):
                     'NAGIOS_STAGING_SEND_BOT': None,
                     'NAGIOS_STAGING_RECEIVE_BOT': None,
                     'APNS_CERT_FILE': None,
+                    'APNS_KEY_FILE': None,
```

Comment:
APNS_KEY_FILE is the private key you generate and APNS_CERT_FILE is the SSL certificate you get from Apple. You can read details [here](http://docs.aws.amazon.com/sns/latest/dg/mobile-push-apns.html)